SPECS/librdkafka/librdkafka.spec

```diff
@@ -0,0 +1,110 @@
+Summary:        The Apache Kafka C library
+Name:           librdkafka
+Version:        1.4.0
+Release:        1%{?dist}
+License:        BSD-2-Clause
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          Development/Libraries/C and C++
+URL:            https://github.com/edenhill/librdkafka
+Source0:        librdkafka-%{version}.tar.gz
+BuildRequires:  cyrus-sasl
+BuildRequires:  gcc >= 4.1
+BuildRequires:  libstdc++-devel
+BuildRequires:  openssl-devel
+BuildRequires:  python2
+BuildRequires:  zlib-devel
+
+%description
+librdkafka is the C/C++ client library implementation of the Apache Kafka protocol, containing both Producer and Consumer support.
+
+%package -n     %{name}%{soname}
+Summary:        The Apache Kafka C library
+Group:          Development/Libraries/C and C++
+Requires:       cyrus-sasl
+Requires:       libstdc++
+Requires:       openssl
+Requires:       zlib
+
+%description -n %{name}%{soname}
+librdkafka is the C/C++ client library implementation of the Apache Kafka protocol, containing both Producer and Consumer support.
+
+%package -n     %{name}-devel
+Summary:        The Apache Kafka C library (Development Environment)
+Group:          Development/Libraries/C and C++
+Requires:       %{name}%{soname} = %{version}
+
+%description -n %{name}-devel
+librdkafka is the C/C++ client library implementation of the Apache Kafka protocol, containing both Producer and Consumer support.
+This package contains headers and libraries required to build applications
+using librdkafka.
+
+%prep
+%setup -q
+
+# --install-deps will install missing dependencies that are not available
+# through BuildRequires, such as libzstd, which will be linked statically.
+%configure --install-deps --disable-lz4-ext
+
+%build
+cat config.log
+make
+examples/rdkafka_example -X builtin.features
+
+%install
+DESTDIR=%{buildroot} make install
+
+%clean
+rm -rf %{buildroot}
+
+%post   -n %{name}%{soname} -p /sbin/ldconfig
+%postun -n %{name}%{soname} -p /sbin/ldconfig
+
+%files -n %{name}%{soname}
+%defattr(444,root,root)
+%{_libdir}/librdkafka.so.%{soname}
+%{_libdir}/librdkafka++.so.%{soname}
+%defattr(-,root,root)
+%doc %{_docdir}/librdkafka/README.md
+%doc %{_docdir}/librdkafka/CONFIGURATION.md
+%doc %{_docdir}/librdkafka/INTRODUCTION.md
+%doc %{_docdir}/librdkafka/STATISTICS.md
+%license %{_docdir}/librdkafka/LICENSE
+%doc %{_docdir}/librdkafka/LICENSES.txt
+
+%defattr(-,root,root)
+#%{_bindir}/rdkafka_example
+#%{_bindir}/rdkafka_performance
+
+%files -n %{name}-devel
+%defattr(-,root,root)
+%{_includedir}/librdkafka
+%defattr(444,root,root)
+%{_libdir}/librdkafka.a
+%{_libdir}/librdkafka.so
+%{_libdir}/librdkafka++.a
+%{_libdir}/librdkafka++.so
+%{_libdir}/pkgconfig/rdkafka++.pc
+%{_libdir}/pkgconfig/rdkafka.pc
+%{_libdir}/pkgconfig/rdkafka-static.pc
+%{_libdir}/pkgconfig/rdkafka++-static.pc
+
+%changelog
+* Mon Jan 04 2021 Henry Li <lihl@microsoft.com> - 1.2.0-1
+- Initial CBL-Mariner import (license: BSD-2-Clause).
```

Comment:
Please indicate the source of the import as well. Are we sure the spec file is released under the BSD license? It doesn't have to match the `License` tag, which informs us about the source's licensing, not the spec file itself, thus the question.

Reply:
The spec file is imported from the librdkafka source code repo: https://github.com/edenhill/librdkafka. I'm unclear about what's the convention to indicate the source of the import for this specific case.

Reply:
We approach each new source of specs individually, or at least I don't think we've developed any streamlined process, since this is uncommon. If the project provides the spec file itself, then I believe the project's and spec's licenses are the same thing, in which case what you have right now (mention of the author and the license info) sounds reasonable. I would only double-check with Jon.

---
In reply to: [555491220](https://github.com/microsoft/CBL-Mariner/pull/523#discussion_r555491220) [](ancestors = 555491220)